src/main/java/frc/robot/constants/Constants2023.java

```diff
@@ -100,16 +100,6 @@ public class Constants2023 extends GameConstants {
     public static final double CLIMBER_BALANCE_THRESH=30;
     public static final double CLIMBER_TIMEOUT_S=10;
 
-    //Intake
-    public static final int INTAKE_MOTOR_1_ID = 111;
-    public static final int INTAKE_MOTOR_2_ID = 12;
```

Comment:
It looks like several INTAKE constants were removed from the Constants2023 file. I don't know if this is a merge issue or not. To test, in Constants.java, extends Constants2023, then try to build the code. If there are missing constants entries the code will not compile. If they have moved to other files correctly (like moving to GameConstants), the code will compile.